ManagedClientConsoleAppSample/Program.cs

```diff
@@ -78,6 +80,7 @@ private static void ListProjects(AuthenticationHeaderValue authHeader)
                 {
                     Console.WriteLine("\tSuccesful REST call");
                     Console.WriteLine(response.Content.ReadAsStringAsync().Result);
+                    Thread.Sleep(10000);
```

Comment:
We don't want to include code that's not essential as it's common for users to copy/paste and use as is.